### ISSUE TITLE
Launch: Needs some visual tweaks to align with reskin

### ISSUE THREAD

**Issue by ollierozdarz** [Automattic/wp-calypso#53973]
### Why
While the Calypso launch page wasn't in scope for our signup reskin launch, we can apply a few visual treatments to get it consistent with a minor effort (and no full design).

### What
With a few visual treatments, we want to improve the UI/UX of the reskinned Calypso Launch page.

We want to: 
- Left align the heading and subheading text. [screenshot](https://cloudup.com/co2EBZwZrvq).
- Change the color of the currency symbol to be the same as the price. [screenshot](https://cloudup.com/cDZj3ATwr0e).
- Align the step counter with the actual number of steps. While there are 2, the counter shows 3 steps! [screenshot](https://cloudup.com/cTTosg16oqA).
- Change the subtext on the launch plan page from "Start with a free site" to "Continue with your free site". [screenshot](https://cloudup.com/c_LdVPS2aI4). **NOTE that the copy should remain the same on the signup flow plan page!**

### Acceptance criteria
- [ ] Heading & Subheading are aligned to the left (like the reskinned domain picker).
- [ ] Currency symbol color is the same as the price.
- [x] The step counter aligns with the **actual** number of steps. @razvanpapadopol in https://github.com/Automattic/wp-calypso/pull/54318
- [ ] Launch Plan Page: Subtext is changed to "Continue with your free site".
- [ ] Signup Flow Plan Page: Subtext remains "Start with a free site".

### Context:
Mentioned in p1624462241024000-slack-C01VA100LEA, together with https://github.com/Automattic/wp-calypso/issues/53975

**Comment by ollierozdarz** [comment]
**Improvements:**

- Left align the heading and subheading text ([screenshot](https://cloudup.com/co2EBZwZrvq)).
- As Niranjan mentions, the currency symbol is green when it makes sense to be the same color ([screenshot](https://cloudup.com/cDZj3ATwr0e)).
- As Niranjan metnions, the step counter seems to be wrong, it shows “1 of 3" and “2 of 3” when there are only two steps ([screenshot](https://cloudup.com/cTTosg16oqA)).
- Launch Plan page: Change the subtext from "Start with a free site" to “Continue with your free site" (but leave the text the same for the signup flow). Niranjan made this suggestion and it makes sense given the user is at the end of their activation journey ([screenshot](https://cloudup.com/c_LdVPS2aI4)).

**Comment by StefanNieuwenhuis** [comment]
@ollierozdarz, @niranjan-uma-shankar  I'm unable to reproduce this (see screenshot). Can you tell me how to reproduce it?
 
<img width="1009" alt="Screenshot 2021-06-24 at 16 37 31" src="https://user-images.githubusercontent.com/12104589/123282782-18a8bc00-d50b-11eb-95ab-253acf65ac07.png">

**Comment by niranjan-uma-shankar** [comment]
Maybe it's seen only on INR currency? Could you please check with proxying through India?

<img width="1261" alt="Screenshot 2021-06-24 at 7 45 27 PM" src="https://user-images.githubusercontent.com/1269602/123293143-cbd1df00-d524-11eb-9acc-38bc27442bc0.png">

**Comment by niranjan-uma-shankar** [comment]
Looks like this issue appears only for those currencies where we offer a first year discount - INR and PHP. Proxying through India or Philippines should help reproduce the issue:

<img width="1199" alt="Screenshot 2021-06-24 at 7 47 41 PM" src="https://user-images.githubusercontent.com/1269602/123293600-2ff4a300-d525-11eb-9370-8c50b59b9c2b.png">

**Comment by StefanNieuwenhuis** [comment]
> Looks like this issue appears only for those currencies where we offer a first year discount - INR and PHP. Proxying through India or Philippines should help reproduce the issue:
> 
> <img alt="Screenshot 2021-06-24 at 7 47 41 PM" width="1199" src="https://user-images.githubusercontent.com/1269602/123293600-2ff4a300-d525-11eb-9370-8c50b59b9c2b.png">

Thanks, @niranjan-uma-shankar! Proxying did the trick. Now I'm able to address the issue and I've learned tons about the proxy options 👍

**Comment by StefanNieuwenhuis** [comment]
> As Niranjan mentions, the step counter seems to be wrong, it shows “1 of 3" and “2 of 3” when there are only two steps.

While looking [at the code](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/config/flows-pure.js#L312), we actually have 3 steps in this flow. 
**Steps**
1. Domain Picker.
2. Plan picker.
3. Launching your site (see screenshot).

Although it might look a bit funny/unnatural, the total number of steps is correct.
<img width="698" alt="Screenshot 2021-06-25 at 14 26 39" src="https://user-images.githubusercontent.com/12104589/123424606-5d8f2a00-d5c1-11eb-9ccf-90b73bca4ca9.png">

**Comment by niranjan-uma-shankar** [comment]
Ah yes, the third step "launching your site" is a hidden step and there's no UI attached to it. It only fires an API request to launch the site, so from the user's perspective there are only two steps to complete. I think this bug's been around even with the blue launch flow and not introduced by the reskin flow. 

On a related note, the "Step 1 of 1" looks a bit off if there's only one step cc @ollierozdarz @StefanNieuwenhuis 

<img src="https://user-images.githubusercontent.com/12104589/123424606-5d8f2a00-d5c1-11eb-9ccf-90b73bca4ca9.png" />

**Comment by ollierozdarz** [comment]
> On a related note, the "Step 1 of 1" looks a bit off if there's only one step cc @ollierozdarz @StefanNieuwenhuis

@StefanNieuwenhuis @niranjan-uma-shankar yeah, I don't think it's going to detract from the page if we were to hide the 'Step 1 of 1' there (above). Even for `/new` when we have 2-3 steps in the processing state, they're so fast (**and** have a progress indicator) that I think hiding the number wouldn't cause any negative movement in metrics.

**Comment by StefanNieuwenhuis** [comment]
> > On a related note, the "Step 1 of 1" looks a bit off if there's only one step cc @ollierozdarz @StefanNieuwenhuis
> 
> @StefanNieuwenhuis @niranjan-uma-shankar yeah, I don't think it's going to detract from the page if we were to hide the 'Step 1 of 1' there (above). Even for `/new` when we have 2-3 steps in the processing state, they're so fast (**and** have a progress indicator) that I think hiding the number wouldn't cause any negative movement in metrics.

Thanks @ollierozdarz, I created https://github.com/Automattic/wp-calypso/issues/54118 to keep track.
What about https://github.com/Automattic/wp-calypso/issues/53973#issuecomment-868503576?

**Comment by ollierozdarz** [comment]
> What about #53973 (comment)?

@StefanNieuwenhuis is the third step, 'Launching your site ', simply the screenshot you posted?

I.e.
![image](https://user-images.githubusercontent.com/67085494/123890976-85083d00-d99b-11eb-8d11-d8897c2a7048.png)

**Comment by StefanNieuwenhuis** [comment]
> @StefanNieuwenhuis is the third step, 'Launching your site ', simply the screenshot you posted?


@ollierozdarz, it is :)

**Comment by richtabor** [comment]
Closing as this is outdated. Thanks.
